docs/advanced/runtime/docs/coop-suspend.md

```diff
@@ -68,6 +68,8 @@ states:
 - Blocking: The current thread is executing code that won't touch managed memory, so it can be assumed to be suspended
 - Blocking and Suspended: The current thread finished executing blocking code but there was a pending suspend against it, it's waiting to be resumed
 
+![Coop state machine transition diagram](coop-state-machine.png)
```

Comment:
You need to put the link as `/docs/advanced/runtime/docs/coop-state-machine.png`, because the coop-suspend.md gets turned into coop-suspend/index.html so the relative link would search in that directory and not find the image there.